Frontend/src/libs/updateUserProfile.tsx

```diff
@@ -0,0 +1,27 @@
+
+export default async function updateUserProfile(
+    name : string ,
+    password : string ,
+    token : string ,
+) {
+    const response = await fetch(
+      //`https://job-fair-frontend-but-backend.vercel.app/auth/update`,
+      "http://localhost:5000/auth/update",
```

Comment:
don't use localhost please

Reply:
It not update test with new API that I give you in discord deployment

Reply:
ให้ chatgpt ช่วย genarate มา


The issue is likely related to Cross-Origin Resource Sharing (CORS) policies. When you try to make a request from your client application (running on a different domain or origin) to the "https://modlangtum-api.vercel.app/auth/update" API endpoint, the browser's security mechanism blocks the request due to CORS restrictions.
CORS is a security feature implemented by web browsers to prevent web applications from making requests to a different domain than the one from which the application was served. This is a security measure to prevent unauthorized access to resources from other domains.
When you use the "http://localhost:5000/auth/update" URL, the request is allowed because it originates from the same domain as the server (localhost).
To solve this issue when making requests to the "https://modlangtum-api.vercel.app/auth/update" endpoint, the server hosting the API needs to configure CORS properly to allow cross-origin requests from your client application's domain.
Here are a few potential solutions:
Configure CORS on the server: If you have access to the server hosting the API (in this case, the Vercel deployment), you need to configure CORS to allow requests from your client application's domain. This typically involves setting the appropriate CORS headers in the server's response. The specific steps depend on the server framework or library you're using.
Use a proxy server: If you don't have direct access to configure CORS on the server hosting the API, you can set up a proxy server on your local development environment or on the server hosting your client application. The proxy server will forward the requests from your client application to the API server, effectively bypassing the CORS restrictions. This approach can be more complex to set up and maintain.
Use a serverless function or middleware: Another option is to create a serverless function or middleware that acts as a proxy between your client application and the API server. This function or middleware can handle CORS and forward the requests to the API server. This approach can be implemented using services like Vercel Functions, AWS Lambda, or other serverless platforms.
Without proper CORS configuration or a suitable workaround, your client application will not be able to make direct requests to the "https://modlangtum-api.vercel.app/auth/update" API endpoint due to the browser's security restrictions.

Reply:
งั้นแก้แล้ว merge มาเลย

Reply:
ไม่รู้แก้ไง ToT

Reply:
let me do it

Reply:
ไม่ต้องแยกละ นอนพนค่อยว่ากันตอนเช้า